src/v/model/metadata.h

```diff
@@ -244,7 +245,11 @@ struct topic_namespace_view {
     friend std::ostream& operator<<(std::ostream&, const topic_namespace_view&);
 };
 
-struct topic_namespace {
+struct topic_namespace
+  : public serde::
```

Comment:
I think we do not need versioning overhead to `topic_namespace` as this will probably never change.

Reply:
Thank you! I changed it to unversioned.